main.go

```diff
@@ -98,32 +100,38 @@ func main() {
 	select {}
 }
 
-// getClients builds and returns both Kubernetes and Calico clients.
-func getClients(kubeconfig string) (*kubernetes.Clientset, *client.Client, error) {
+// getClients builds and returns Kubernetes, Calico and Extensions clients.
+func getClients(kubeconfig string) (*kubernetes.Clientset, *client.Client, *rest.RESTClient, error) {
 	// First, build the Calico client using the configured environment variables.
 	cconfig, err := client.LoadClientConfig("")
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 
 	// Get Calico client
 	calicoClient, err := client.New(*cconfig)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to build Calico client: %s", err)
+		return nil, nil, nil, fmt.Errorf("failed to build Calico client: %s", err)
 	}
 
 	// Now build the Kubernetes client, we support in-cluster config and kubeconfig
 	// as means of configuring the client.
 	k8sconfig, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to build kubernetes client config: %s", err)
+		return nil, nil, nil, fmt.Errorf("failed to build kubernetes client config: %s", err)
 	}
 
 	// Get kubenetes clientset
 	k8sClientset, err := kubernetes.NewForConfig(k8sconfig)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to build kubernetes client: %s", err)
+		return nil, nil, nil, fmt.Errorf("failed to build kubernetes client: %s", err)
 	}
 
-	return k8sClientset, calicoClient, nil
+	// Get extensions client
+	extensionsClient, err := k8s.BuildExtensionsClientV1(*k8sconfig)
```

Comment:
Just noticed this, should have spotted in the libcalico PR.

`BuildExtensionsClientV1` is the wrong name for this function.  It should be `v1beta1` or just `BuildExtensionsClient`.

Probably not worth changing at this point given it's temporary anyway.....

Reply:
will probably leave this as is for now, but will fix up if we need to rev libcalico-go